Makefile

```diff
@@ -111,7 +111,7 @@ deploy: manifests
 
 load-image: ## Load the operator image to a kind cluster
 load-image: container
-	./hack/load-image.sh $(IMAGE) $(VERSION)
+	./hack/load-image.sh $(IMAGE) $(OLD_VERSION) $(VERSION)
```

Comment:
Did these image flow changes get pulled in from another PR?

Reply:
@youngnick not a separate PR, but a separate commit. The image flow changes fix a regression introduced by https://github.com/projectcontour/contour-operator/commit/1410e5952edb1526664319051955a376330657bd and the ConfigMap changes fix a regression introduced by https://github.com/projectcontour/contour/pull/3481. Both commits are needed for CI to pass and why they are included together. I can split if needed but one of the PRs will need to get merged with CI failing. Thoughts?

Reply:
Ah, that's okay then. Let's concentrate on getting this in.

Reply:
I think in the future it'd be easier for reviewers if they were in separate PRs, but will proceed as-is on this one.

Reply:
@skriss thanks for the review. I'm usually against having a PR resolve separate concerns. However, neither commit will pass CI on its own, so I thought using separate commits was the best way to proceed.